src/main/scala/cognite/spark/v1/FlexibleDataModelRelationUtils.scala

```diff
@@ -225,7 +225,7 @@ object FlexibleDataModelRelationUtils {
       row: Row): Either[CdfSparkException, DirectRelationReference] =
     Try {
       val struct = row.getStruct(schema.fieldIndex(propertyName))
-      val space = Option(struct.getAs[Any]("space"))
+      val space = Option(struct.getAs[Any]("spaceExternalId"))
```

Comment:
I would consider handling both `space` & `spaceExternalId`.
If `space` is not defined try `spaceExternalId` as a fall back.
That make more sense than changing everything to `spaceExternalId`.
All the other places use `space` to be be consistent with the api spec.

Reply:
I don't think it's a good idea to support both here, we can support space to keep the existing tests works for v3 with both syntax `spaceExternalId` and `space` in the named_struct, but it's useless since the UDF in jetfire-backend continues to generate `spaceExternalId` anyway. It is redundant and doesn't bring much value here. If we keep `spaceExternalId`, you can rename it to space when deprecating v2 in both Spark DS and jetfire. This is an internal naming in the code, we don't expose it to any user so we can live with the fact it's not consistent with the api spec for v3, in v2 api spec it is still spaceExternalId